packages/wrangler/src/repl.ts

```diff
@@ -0,0 +1,18 @@
+import { Miniflare } from "miniflare";
+import { logger } from "./logger";
+
+export async function repl() {
+	logger.log("Interactive REPL session started though Miniflare");
+	const mf = new Miniflare({
```

Comment:
I guess this will depend on the use case for how exactly we want to implement this - do you have any idea of the primary reasons people have been using repl in miniflare thus far?

Reply:
> I guess this will depend on the use case for how exactly we want to implement this - do you have any idea of the primary reasons people have been using repl in miniflare thus far?

Either way we wouldn't want the REPL experience to be too different from the `--local` Dev experience.